fabric/bin/peer.sh

```diff
@@ -254,6 +254,12 @@ handle_channel_join() {
     # - call real peer so channel is joined
     try $RUN ${FABRIC_BIN_DIR}/peer "${ARGS_EXEC[@]}"
 
+    # - ask tlcc to join channel
+    #   IMPORTANT: right now a join is _not_ persistant, so on restart of peer,
+    #   it will re-join old channels but tlcc will not!
```

Comment:
given that you already moved it (see below) and shows up as a diff for the whole section: should we maybe extend the comment that (presumably) manually calling the JOIN_CHANNEL after a peer restart presumably should make it work, though?

Reply:
moved it back where it was